PR_python.py

```diff
@@ -1,4 +1,6 @@
 print("Hello World")
 
 for i in range(5):
-    print(i)
\ No newline at end of file
+    print(i)
+
+print("Brijendra gurjar is learning")
```

Comment:
@Brijcodes Could you please explain more about this line?

Reply:
@Brijendra07 This is just a test print statement.